modal/punctuator/__init__.py

```diff
@@ -75,16 +72,12 @@ def transform(self, input_str: str):
         import threading
         import time
 
-        output = [
-            None
-        ]  # Use a list to hold the output to bypass Python's scoping limitations
```

Comment:
these were just `ruff format` changes -- only running on modified files through the pre-commit hook

not 100% sure the formatter is running in CI tho :thinking: might need to check that in another PR & just do a single global format so its fixed going forward

Reply:
Ooh - I also forgot about the line-length. Kinda want to keep it at 80 since I split my screen a lot :d

Reply:
@louisgv heh no worries :) changed it back to 80 here 01e4c12

sometimes i forget how small i keep my font size compared to others

once this pr lands i'll make a followup that actually does a global format & enforces ruff format in ci